### PR TITLE
adding Beagle framework

### PR DESCRIPTION
**Pull request by caroldf07**
I think my project would be a great fit for Up for Grabs.

Let me know if there's any feedback you have about the template!

### CODE REVIEW

File: _data/projects/beagle-framework.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - front-end
+  - flutter
+  - ios
+  - reactive native
+  - android
+upforgrabs:
+  name: good first issue
+  link: https://github.com/ZupIT/beagle/labels/good%20first%20issue
+stats:
+  issue-count:
```
Comment (ritwik12) [inline _data/projects/beagle-framework.yml]: These lines are not required, please remove.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Review by ritwik12 — APPROVED** [review]

File: _data/projects/beagle-framework.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+site: https://usebeagle.io/
+tags:
+  - mobile
+  - kotlin
+  - back-end
+  - web
+  - front-end
+  - flutter
+  - ios
+  - reactive native
```
Comment (ritwik12) [inline _data/projects/beagle-framework.yml]: @caroldf07 No space allowed too, also did you meant react-native?

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@caroldf07 Sorry for the trouble but we have kept the tag names generic so that we won't have duplicate ones.

**Comment by caroldf07** [comment]
@ritwik12 it's ok, the mistake was mine :)

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@caroldf07 Thanks for adding your projects and fixing the issues promptly.

**Pull request merged**